src/constants.js

```diff
@@ -25,7 +25,7 @@ const LINK_MIN_OFFSET = 25
 const LINK_RANDOM = 100
 const LINK_STRENGHT = 0.3
 const MAX_SCREEN_NODES = 25
-const MARGIN = 37
+const MARGIN = 100
```

Comment:
bumping up this margin because the nodes were hitting the timeline (on the bottom) and going off screen (on the top).